utils/config.go

```diff
@@ -232,24 +238,13 @@ func LoadConfig(fileName string) {
 	config := Config{}
 	err = decoder.Decode(&config)
 	if err != nil {
-		panic("Error decoding configuration " + err.Error())
-	}
-
-	// Check for a valid email for feedback, if not then do feedback@domain
-	if _, err := mail.ParseAddress(config.EmailSettings.FeedbackEmail); err != nil {
-		l4g.Error("Misconfigured feedback email setting: %s", config.EmailSettings.FeedbackEmail)
-		config.EmailSettings.FeedbackEmail = "feedback@localhost"
+		panic("Error decoding config file=" + fileName + ", err=" + err.Error())
 	}
 
 	configureLog(config.LogSettings)
 
 	Cfg = &config
 	SanitizeOptions = getSanitizeOptions()
-
-	// Validates our mail settings
-	if err := CheckMailSettings(); err != nil {
-		l4g.Error("Email settings are not valid err=%v", err)
-	}
```

Comment:
I assume you removed the validation here because it will be done in the admin console.